lib/CommandImplementations.ts

```diff
@@ -542,6 +542,62 @@ function formatVaccineData(data: any, population: number, embed: MessageEmbed) {
     );
 }
 
+export async function handleStock(msg: Message, args: string[]) {
+    if (args.length === 0) {
+        msg.channel.send("You need to include a ticker. Example: $stock IBM");
+        return;
+    }
+
+    const [ ticker ] = args;
+
+    const res = await fetch(`https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=${ticker.toUpperCase()}&apikey=${config.stockApiKey}`);
+    const stockData = await res.json();
+
+    const embed = new MessageEmbed()
+    .setColor(Number(stockData['Global Quote']['09. change']) < 0 ? '#C8102E' : '#00853D')
+    .setTitle(ticker.toUpperCase())
+    // .setThumbnail('https://i.imgur.com/FnbQwqQ.png')
+    .addFields(
+        {
+            name: 'Price',
+            value: `$${numberWithCommas(stockData['Global Quote']['05. price'])}`,
```

Comment:
We should probably call `roundToNPlaces(val, 2)` for all the values in this embed to cap them at a sensible length.
You might want to alias the formatting, to keep it short, e.g.

```js
const f = (val) => numberWithCommas(formatToNPlaces(val, 2));
```